pkgs/top-level/all-packages.nix

```diff
@@ -23938,6 +23938,14 @@ in
 
   chocolateDoom = callPackage ../games/chocolate-doom { };
 
+  clonehero-unwrapped = pkgs.callPackage ../games/clonehero { };
+
+  clonehero-xdg-wrapper = pkgs.callPackage ../games/clonehero/xdg-wrapper.nix { };
+
+  clonehero-fhs-wrapper = pkgs.callPackage ../games/clonehero/fhs-wrapper.nix { };
```

Comment:
Do all of these need to be in the top-level namespace?

Reply:
Probably not, I added them when I was testing the different layers, but I can remove them.